### PR TITLE
[fv_lekwungen] new keyboard

### PR DESCRIPTION
**Pull request by xyzhelen**
New desktop and mobile keyboard for Lekwungen (Lək̓ʷəŋən).

### CODE REVIEW

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by LornaSIL** [comment]
@xyzhelen are you wanting to update the language tag or go ahead and get this approved?

**Comment by xyzhelen** [comment]
> @xyzhelen are you wanting to update the language tag or go ahead and get this approved?

Hi, can we go ahead with this? I'll discuss the language tag with others and it might go in a future revision

**Comment by LornaSIL** [comment]
Please include `LICENSE.md` in the list of files in the .kps and then include it on the **Details** tab for the LICENSE.md.
Hopefully then it will be good to go.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

File: release/fv/fv_lekwungen/source/fv_lekwungen.kps [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      <Description>Keyboard Lekwungenm</Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.kmx</FileType>
+    </File>
+    <File>
+      <Name>help\fv_lekwungen.php</Name>
+      <Description>File fv_lekwungen.php</Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.php</FileType>
+    </File>
```
Comment (darcywong00) [inline release/fv/fv_lekwungen/source/fv_lekwungen.kps:58]: This is online help and can be removed from the package source
```suggestion
```

**Comment by LornaSIL** [comment]
Actually, that file should not be included in the .kps.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Review by LornaSIL — APPROVED** [review]
I think it's good.

**Pull request merged**